e107_plugins/login_menu/login_menu_class.php

```diff
@@ -117,7 +117,7 @@ function get_external_list($sort = true)
     function parse_external_list($active=false, $order=true)
 	{
         //prevent more than 1 call
-        if(($tmp = e107::getRegistry('loginbox_elist')) !== FALSE) return $tmp;
+        if(($tmp = e107::getRegistry('loginbox_elist', false)) !== FALSE) return $tmp;
```

Comment:
Although this usage of the default value will technically produce the right result, the type of `false` causes a failing type check because https://github.com/e107inc/e107/commit/901985ea80a42607af69222e8776fce89d8fa578#diff-230529b78667c0efa3482b07aa9cabe79b2fdb03135969acf9f178b34685f4a9R521 established that the second argument must be `null`.

I recommend changing the usage to:

```php
if(($tmp = e107::getRegistry('loginbox_elist')) !== null)
```

Reply:
Thanks.